pfioh/mount_dir.py

```diff
@@ -37,6 +37,14 @@ def storeData(self, **kwargs):
             if k == 'is_zip': b_zip                 = v
             if k == 'd_ret': d_ret                  = v
         
+        if str_destPath == '':
```

Comment:
Unless we are worried about str_desPath not being a string, you should be able to use `if str_destPath` since empty string is false.